.editorconfig

```diff
@@ -17,3 +17,7 @@ trim_trailing_whitespace = false
 [Makefile]
 indent_style = tab
 indent_size = 4
+
+[{*.go,go.mod,go.sum}]
+indent_style = tab
+indent_size = 8
```

Comment:
That's a HUGE number!

Reply:
I restored this value to `4`, sorry, I directly quoted： https://github.com/editorconfig/editorconfig-core-go/blob/master/.editorconfig

Reply:
4 is good!

BTW: my favorite is 2, and I believe that's the most common style in the JS/TS community.